### PR TITLE
CI target for Makefile

### PR DESCRIPTION
**Pull request by LK4D4**
Seems like sign checking better to do with jfrazelle/leeroy as for docker projects.

### CODE REVIEW

File: script/.validate [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -z "$VALIDATE_UPSTREAM" ]; then
+	# this is kind of an expensive check, so let's not do this twice if we
+	# are running more than one validate bundlescript
+	
+	VALIDATE_REPO='https://github.com/docker/swarm.git'
```
Comment (tianon) [inline script/.validate]: Pretty sure you meant `https://github.com/opencontainers/runc.git` here. :wink:

Comment (LK4D4) [inline script/.validate]: Indeed, forgot to commit :)

**Comment by mrunalp** [comment]
LGTM

**Comment by jfrazelle** [comment]
do you all want leeroy because i can set it up

**Comment by mrunalp** [comment]
@jfrazelle I don't know leeroy is but some CI will be great :) Thanks!

**Comment by jfrazelle** [comment]
so someone just needs to give permissions to @gordontheturtle for the repo

**Comment by jfrazelle** [comment]
and then add a couple webhooks and we will be in action

**Comment by mrunalp** [comment]
We need @crosbymichael for that.

**Comment by crosbymichael** [comment]
@jfrazelle I added gordon but he will have to accept the organization invite first.

**Comment by jfrazelle** [comment]
I will tell him ;)
(I see the email haha)

File: Makefile [hunk 2]
```diff
@@ -25,3 +27,9 @@ install:
 
 clean:
 	rm runc
+
+validate: vet
+	script/validate-gofmt
+	go vet ./...
+
+ci: validate test
```
Comment (jfrazelle) [inline Makefile]: can this be localtest instead because I will already have it running in a container :)

**Comment by LK4D4** [comment]
@jfrazelle I changed to localtest

**Comment by jfrazelle** [comment]
sweet LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**